x-pack/plugins/apm/public/components/app/service_map/popover/service_contents.tsx

```diff
@@ -43,7 +43,11 @@ export function ServiceContents({
   const nodeData = elementData as NodeDataDefinition;
   const apmRouter = useApmRouter();
 
-  const { query } = useApmParams('/*');
+  const { query } = useAnyOfApmParams(
```

Comment:
Is this change related to diagnostics tool?

Reply:
This is to make sure that that this doesn't check params for any path. It should only check params for the paths where the service map is rendered. 

Without this change it would complain that `comparisonEnabled` and `offset` were optional (they are not required on the diagnostics paths). 

Slightly off topic: What's surprising to me is that this only became a problem when I added definitions for params like `rangeFrom`. It seems like the type check excludes paths without param entirely (which it shouldn't). Any idea why this is @dgieselaar ?

Reply:
No, sounds like a bug tbh